ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,134 @@
 ﻿namespace ConsoleApp1
 {
-	internal class Program
-	{
-		static void Main(string[] args)
-		{
-		}
-	}
-}
\ No newline at end of file
+    internal class Program
+    {
+
+
+        static void Main(string[] args)
+        {
+            string str1 = "The quick brown fox jumps over the lazy dog";
+            string str3 = "4.4.315.4.1045";
+            string str4 = "#@₴?$0";
+            string str5 = "AaBbCcDdEeFf";
+            string str6 = "Mississippi";
+            char[] temp = str1.ToCharArray();
+            Array.Reverse(temp);
+            string str2 = new string(temp);
+            Console.WriteLine(str1 + " " + str2);
+
+            Console.WriteLine(Compare(str1, str2));
+            Console.WriteLine(Compare(str1, str1));
+
+            Console.WriteLine("Letters, digits, symbols: ");
+
+            Console.WriteLine(str1);
+            Analyze(str1);
+            Console.WriteLine(Sort(str1));
+            Console.WriteLine(Duplicate(str1));
+            Console.WriteLine();
+
+
+            Console.WriteLine(str2);
+            Analyze(str2);
+            Console.WriteLine(Sort(str2));
+            Console.WriteLine(Duplicate(str2));
+            Console.WriteLine();
+
+            Console.WriteLine(str3);
+            Analyze(str3);
+            Console.WriteLine(Sort(str3));
+            Console.WriteLine(Duplicate(str3));
+            Console.WriteLine();
+
+            Console.WriteLine(str4);
+            Analyze(str4);
+            Console.WriteLine(Sort(str4));
+            Console.WriteLine(Duplicate(str4));
+            Console.WriteLine();
+
+            Console.WriteLine(str5);
+            Analyze(str5);
+            Console.WriteLine(Sort(str5));
+            Console.WriteLine(Duplicate(str5));
+            Console.WriteLine();
+
+            Console.WriteLine(str6);
+            Analyze(str6);
+            Console.WriteLine(Sort(str6));
+            Console.WriteLine(Duplicate(str6));
+
+            Console.WriteLine();
+
+
+        }
+        static bool Compare(string str1, string str2)
+        {
+            if (str1 == str2)
```

Comment:
You need to implement compare each character from string str1 with each character from str2